### DISCUSSION TITLE
how to add redirect flag in .args

### DISCUSSION THREAD

**Discussion by badiku** [jart/cosmopolitan#689] [General]
why not work?

> -r /X=/Y 	redirect X to Y [repeatable]
> -R /X=/Y  rewrites X to Y [repeatable]

I add this in .args:

```
-R /help=/help.txt
-w/help
...
```

but got error "404 Not Found ".

if I run command ` redbean.com -R /help=/help.txt` then it is ok.

can I add redirect in .args?

**Comment by pkulchenko** [comment]
I agree that it should work, but it doesn't. I took a quick look, but can't figure out why exactly it's not working. I plan to take another look in few days.
